compiler/rustc_middle/src/ty/closure.rs

```diff
@@ -194,15 +194,15 @@ impl<'tcx> CapturedPlace<'tcx> {
 
 #[derive(Copy, Clone, Debug, HashStable)]
 pub struct ClosureTypeInfo<'tcx> {
-    user_provided_sig: ty::CanonicalPolyFnSig<'tcx>,
+    user_provided_sig: &'tcx ty::CanonicalPolyFnSig<'tcx>,
```

Comment:
why this change?

Reply:
type sizes get larger so it makes some query machinery complain